119-namespaced-conditionalpanel-demo/tests/testthat/tests.R

```diff
@@ -0,0 +1,15 @@
+library(testthat)
+library(shiny)
+library(withr)
+
+# TODO Determine the ideal way for module-reliant test code to load an app
+with_dir(shiny:::findApp(), {
+  local({
+    source("app.R")
```

Comment:
Is this still needed given you are passing in `env`?

Reply:
You know, I think it actually is, because of this bug that I just filed: https://github.com/rstudio/shiny/issues/2746

Reply:
Thanks for pointing that out though, I had forgotten why it was necessary.